Test scripts/Ilse/bulk-poli-temp-to-word.vbs

```diff
@@ -73,17 +94,33 @@ Do
         err_msg = ""
         dialog Dialog1
         cancel_without_confirmation
-        If ButtonPressed = select_a_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
-        If trim(file_selection_path) = "" then err_msg = err_msg & vbcr & "* Select a file to continue."
-        If err_msg <> "" Then MsgBox err_msg
+        'Initial POLI TEMP List Excel file
+        If ButtonPressed = select_initial_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
```

Comment:
In testing I found that if I selected Browse and then selected the file in file explorer, the script would immediately run rather than returning back to the initial dialog and allowing the user to select or confirm the folder where the files should be saved.

Reply:
@MDR22 This is because we have defaulting for X127 users for both the file_selection_path and the root_file_path. I'm going to update this to actually bypass the the dialog for X127 users (really since that's just Jake).

Reply:
I will also create a secondary do....loop to give the out of county users the option to select the folder path in the same dialog.